dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -233,6 +233,7 @@ def test_add_profilers(self, *mocks):
         self.assertEqual(
```

Comment:
I think in the below, we are missing a test when we choose options that disable the count specifically. We would need to ensure the report return values are `None` (I think that's what they would return as).

Reply:
```
"row_has_null_ratio": self._get_row_has_null_ratio(),
"row_is_null_ratio": self._get_row_is_null_ratio(),
```
these two int he report specifically.

Reply:
Currently, 
```
def _get_row_has_null_ratio(self) -> float | None:
        """Return whether row has null ratio."""
        if not self.options.row_statistics.is_enabled:
            ...
```
only returns `None` for row_stats being disabled, but needs to be updated to include the case of null option disabled

Reply:
I think the above still needs to be addressed.

Reply:
looks fixed now